src/app/components/tabs/tabs-demo.component.html

```diff
@@ -11,3 +11,15 @@
     This tab cannot be closed.
   </sky-tab>
 </sky-tabset>
+
+<h3>Tabs example with counter</h3>
```

Comment:
How about we make a change to have a close on one of your demos with the count and then let them decide if they want the two demos consolidated?